jwst/datamodels/schemas/throughput.schema.yaml

```diff
@@ -1,6 +1,6 @@
 title: Filter throughput data model
 allOf:
-- $ref: core.schema.yaml
```

Comment:
Also, there was a typo in the mods applied to throughput.py in #715. Line 16 needs to have "ThroughModel" updated to "ThroughputModel". Please include that update here too.